.storybook/webpack.config.js

```diff
@@ -52,6 +53,7 @@ if (USER_ID && USER_ACCESS_TOKEN) {
       "process.env": {
         USER_ID: JSON.stringify(USER_ID),
         USER_ACCESS_TOKEN: JSON.stringify(USER_ACCESS_TOKEN),
+        USER_LAB_FEATURES: JSON.stringify(USER_LAB_FEATURES),
```

Comment:
injects the ENV var into the special `process.env` sandbox that storybooks uses.